chainerrl/replay_buffer.py

```diff
@@ -402,20 +402,21 @@ def stop_current_episode(self):
 
 def batch_experiences(experiences, xp, phi, batch_states=batch_states):
 
-    return {
+    batch_exp = {
         'state': batch_states(
             [elem['state'] for elem in experiences], xp, phi),
         'action': xp.asarray([elem['action'] for elem in experiences]),
         'reward': xp.asarray(
             [elem['reward'] for elem in experiences], dtype=np.float32),
         'next_state': batch_states(
             [elem['next_state'] for elem in experiences], xp, phi),
-        'next_action': xp.asarray(
-            [elem['next_action'] for elem in experiences]),
         'is_state_terminal': xp.asarray(
             [elem['is_state_terminal'] for elem in experiences],
-
             dtype=np.float32)}
+    if all(elem['next_action'] is not None for elem in experiences):
+        batch_exp['next_action'] = xp.asarray(
+            [elem['next_action'] for elem in experiences])
```

Comment:
Should we check all `next_action`s are `None` otherwise?

Reply:
If some of `elem['next_action']` is None, then `cupy.asarray` fails due to `Value Error: Unsupported dtype object`. So I batch them only if all of them are not None.

Reply:
OK.  I understand the design that a replay buffer makes `elem['next_action']` available if it'll be available as `xp.ndarray`.